Ginger/GingerCore/Actions/ActSubmit.cs

```diff
@@ -97,7 +97,7 @@ ePlatformType IObsoleteAction.GetTargetPlatform()
 
         bool IObsoleteAction.IsObsoleteForPlatform(ePlatformType platform)
         {
-            if (platform == ePlatformType.Web || platform == ePlatformType.Mobile || platform == ePlatformType.NA)
+            if (platform == ePlatformType.Web ||  platform == ePlatformType.NA)
             {
```

Comment:
i asked to remove the Mobile only for GenericElement Action and leave it for rest